packages/lit-dev-content/site/tutorials/content/async-directive/08.md

```diff
@@ -0,0 +1,48 @@
+The directive is now complete. Now let's see how and where we could use it.
+
+In our example element template, we've been rendering the `timeAgo` directive
+into a [child expression](/docs/templates/expressions/#child-expressions) as a
+string:
+
+```ts
+      <p>This page was rendered ${timeAgo(timeCreated)}.</p>
+```
+
+However, our directive can be used in _any_ expression type, including
+[attribute expressions](/docs/templates/expressions/#attribute-expressions) and
+[property expressions](/docs/templates/expressions/#property-expressions). So
+let's say we had a `<comment-card>` element that accepted a few fields like
+`subject`, `description`, and `time`. We've gone ahead and imported one for you,
+so go ahead and add this to the template also:
+
+```ts
+  <comment-card user="litdeveloper"
+                time="12:49 pm"
+                subject="Just tried AsyncDirectives!"
+                content="Cool stuff, they're really powerful!">
+  </comment-card>
+```
+
+Now, since the element accepts time as a string, we can bind the `timeAgo`
+directive to that property, and it will periodically set the element's
+`time` property, without the outer template needing to re-render:
+
+```ts
+  .time=${timeAgo(timeCreated)}
+```
+
+Congratulations! You now have a useful, working `AsyncDirective`!
+
+Ideas for further study:
+* Add an optional `frequency` parameter to the directive, to customize how
+  often the directive re-renders its value. Bonus points for making sure the
+  user can change the frequency after the timer has started!
+* Make an `auto` mode for the directive that scales the `frequency` that the
+  timer updates according to the amount of time that has elapsed.
+* Try making a [component](/docs/components/overview/) version of the `timeAgo`
+  directive. How about a [ReactiveController](/docs/composition/controllers/)
```

Comment:
Suggest removing the discussion of controllers as I think it makes the tutorial end on an unnecessarily confusing/complex note.

Reply:
Huh, I thought this put it more in context...